test/Templates.Test/Helpers/TemplateTestBase.cs

```diff
@@ -132,7 +132,13 @@ protected void RunDotNet(string arguments)
 
         protected void RunDotNetEfCreateMigration(string migrationName)
         {
-            var args = $"ef migrations add {migrationName}";
+            var assembly = typeof(TemplateTestBase).Assembly;
+
+            var DotNetEfFullPath = assembly.GetCustomAttributes<AssemblyMetadataAttribute>()
+                .FirstOrDefault(attribute => attribute.Key == "DotNetEfFullPath")
```

Comment:
Can probably call `.First()` instead of `.FirstOrDefault()` considering that if there's no item in the list, the next line will throw a less meaningful NullRef anyway.